pypit/artrace.py

```diff
@@ -2038,17 +2076,20 @@ def trace_slits(slf, mstrace, det, pcadesc="", maskBadRows=False, min_sqm=30.):
         ww = np.where(edgearr > 0)
         rmin, rmax = np.min(edgearr[ww]), np.max(edgearr[ww])  # min/max are switched because of the negative signs
         #msgs.info("Ignoring any slit that spans < {0:3.2f}x{1:d} pixels on the detector".format(settings.argflag['trace']['slits']['fracignore'], int(edgearr.shape[0]*binby)))
-        msgs.info("Ignoring any slit that spans < {0:3.2f}x{1:d} pixels on the detector".format(settings.argflag['trace']['slits']['fracignore'], int(edgearr.shape[0])))
-        fracpix = int(settings.argflag['trace']['slits']['fracignore']*edgearr.shape[0])
+
+        # JFH I think this should be using the spatial dimension of the image and not the spectral dimension of the image, and am changing it accordingly
```

Comment:
Not true - this is used for when some parts of an order are traced off the detector. This should be changed back to the original